enterprise/admin/forms.py

```diff
@@ -103,6 +121,37 @@ def clean_course(self):
         except (HttpClientError, HttpServerError):
             raise ValidationError(ValidationMessages.INVALID_COURSE_ID.format(course_id=course_id))
 
+    def clean_program(self):
+        """
+        Clean program.
+
+        Try obtaining program treating form value as program UUID or title.
+
+        Returns:
+            dict: Program information if program found
+        """
+        program_id = self.cleaned_data[self.Fields.PROGRAM].strip()
+        if not program_id:
+            return None
+
+        try:
```

Comment:
@e-kolpakov -- just checking -- are you planning on addressing this suggestion or leaving as-is?